compiler/linter/Error/Recover.ml

```diff
@@ -0,0 +1,4 @@
+type recover_flags =
```

Comment:
What is the interest of having a file for that instead of moving in CompileError ?